SConstruct

```diff
@@ -84,9 +85,13 @@ def update_rpm_version(version, tag):
             spec[line_num] = "Release:       {}%{{?relval}}%{{?dist}}\n".\
                              format(release)
         if line == "%changelog\n":
+            rpmdevpackager = find_executable('rpmdev-packager')
+            if not rpmdevpackager:
+                print("No rpmdev-packager found in path.")
+                return False
             try:
                 packager = subprocess.Popen(
-                    'rpmdev-packager', stdout=subprocess.PIPE).communicate(
+                    rpmdevpackager, stdout=subprocess.PIPE).communicate(
```

Comment:
I don't understand the need for this change, but if the command isn't found then the error change as a result of this PR from the verbose output on L97 to the terse output on L90.  The longer error message should be moved to where it'll be helpful, and L97 should say something like "error executing..."

Reply:
The error was due to partial path on 'rpmdev-packager'. the other solution was to apply # nosec on this line.

Reply:
Please see the new commit for the restore subprocess call 'rpmdev-packager'.